service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -128,6 +121,9 @@ public UUID createWorkspace(Workspace workspace, AuthenticatedUserRequest userRe
    * <p>Throws ForbiddenException if the user is not permitted to perform the specified action on
    * the workspace in question.
    *
+   * <p>Throws StageDisabledException if assertMcStage is true and this is not an MC_WORKSPACE stage
```

Comment:
I'm confused, this method doesn't deal with MC_WORKSPACE. validateMcWorkspaceAndAction...

Reply:
whoops, leftover comment from a previous approach